prompttools/playground/playground.py

```diff
@@ -87,7 +92,7 @@
     for j in range(1, instruction_count + 1):
         with cols[j]:
             instructions.append(
-                st.text_area("System Message" if model_type == "OpenAI Chat" else "Instruction", key=f"col_{j}")
+                st.text_area("System Message" if model_type == "OpenAI Chat" else "Instruction", placeholder="You are an AI assistant.", key=f"col_{j}")
```

Comment:
I think instead of `placeholder` you want `value`. `placeholder` will only display, but `value` will set the default value for system messages. That way, if folks don't set the prompt the application will still work.